__docs__/wonder-blocks-popover/popover-content.stories.tsx

```diff
@@ -93,17 +93,15 @@ export const Emphasized: StoryComponentType = {
     render: (args) => <PopoverContent {...args} />,
 };
 
-Emphasized.parameters = {
-    docs: {
-        description: {
-            story:
-                `Text-only variant with added emphasis.\n\n` +
-                `**NOTE:** When using this variant, make sure to apply the \`light\`
-        prop to each button`,
-        },
-    },
-};
-
+/**
+ * Decorate the popover with an illustrated icon. You need to pass an `icon`
+ * prop with the following constraints:
+ * - string: The URL of the icon asset
+ * - `<img>` or `<svg>`: Make sure to define a width
+ *
+ * Use the `iconAlt` prop to provide alternative text for the icon if it
```

Comment:
This is new!

Reply:
Similar to my other comment, maybe it would be good to specify that this will only be applied when the icon is being passed as a `string`.